.storybook/blocks/PropertiesTable.jsx

```diff
@@ -0,0 +1,62 @@
+import "@spectrum-css/table";
+
+import { useOf } from "@storybook/blocks";
+import { ResetWrapper } from "@storybook/components";
+import { styled } from "@storybook/theming";
+import React from "react";
+import { Body, Code, LinkableHeading } from "./Typography.jsx";
+
+export const Table = styled.table`
+	padding-block: 10px;
+`;
+
+/**
+ * Displays the current version number of the component. The version is read from
+ * the component's parameters, where it was sourced from the package.json file.
+ *
+ * Also displays a component status of "deprecated" if it is set in the story's
+ * parameters.
+ *
+ * Usage of this doc block within MDX template(s):
+ *  <PropertiesTable />
+ */
+export const PropertiesTable = () => {
+	const storyMeta = useOf("meta");
+
+	const packageJson = storyMeta?.csfFile?.meta?.parameters?.packageJson ?? {};
+	const metadata = storyMeta?.csfFile?.meta?.parameters?.metadata ?? {};
+
+	if (!packageJson?.name) return;
+	if (!metadata?.modifiers) return;
+
+	return (
+		<ResetWrapper>
+			<LinkableHeading id="mods" size="m">
+				<a aria-hidden="true" href="#mods" tabIndex="-1" target="_self"></a>
```

Comment:
I know Josh mentioned this, but I'm also not getting the `#mods` added to the URL. I don't have a suggestion on how to fix that, but wanted to note it once more.